<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giao diện trang  chủ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Bảng tài nguyên</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="inProcessing">
          <property name="styleSheet">
           <string notr="true">color:red</string>
          </property>
          <property name="text">
           <string>*đang xử lý</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="folderPath">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>D:\Folinas Socials Tool\thumucupload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="choosePath">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadToTable">
          <property name="text">
           <string>Load vào bảng</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bulkScheduleWindowBtn">
          <property name="text">
           <string>Lên lịch hàng loạt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearTable">
          <property name="text">
           <string>Xoá bảng</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openQueue">
          <property name="text">
           <string>Khởi động hàng chờ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="videosTable">
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
        </property>
        <column>
         <property name="text">
          <string>Tiêu đề</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mô tả</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Thẻ tag</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Duration</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Hiển thị</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Lên lịch</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Source</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Upload to</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Đường dẫn</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widgetAbove" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>350</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Bảng tài khoản</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="typeAddComboBox">
             <property name="minimumSize">
              <size>
               <width>102</width>
               <height>27</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #ced6e0;
padding-left: 5px;
border-radius: 5px;
color: #2f3542;</string>
             </property>
             <item>
              <property name="text">
               <string>Facebook</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Youtube</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X - Twitter</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Instagram</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addOneAccount">
             <property name="text">
              <string>Thêm tài khoản</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="profilesTable">
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
           </property>
           <column>
            <property name="text">
             <string>Tên hồ sơ</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Email/Username</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tình trạng</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nền tảng</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Proxy</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widgetRight" native="true">
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Tài sản tài khoản</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="assetsTable">
            <property name="maximumSize">
             <size>
              <width>355</width>
              <height>16777215</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string>Tài sản</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuC_i_t">
    <property name="title">
     <string>Cài đặt</string>
    </property>
    <addaction name="uploadSettingsBtn"/>
   </widget>
   <addaction name="menuC_i_t"/>
  </widget>
  <action name="uploadSettingsBtn">
   <property name="text">
    <string>Cài đặt upload</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
